crates/contrib/pgvector/test/pgvector-regress.sh

```
@@ -0,0 +1,47 @@
+shift $((OPTIND-1))
+
+PGVECTOR_REF=${PGVECTOR_REF:-v0.8.5}
+SRC=${PGVECTOR_SRC:-${TMPDIR:-/tmp}/pgvector-src-$PGVECTOR_REF}
+if [ ! -d "$SRC/test/sql" ]; then
+  git clone -q --depth 1 --branch "$PGVECTOR_REF" https://github.com/pgvector/pgvector.git "$SRC"
+fi
+
+# Tests whose features pgrust does not ship (bit opclass, ivfflat).
+SKIP_RE='^(bit|hnsw_bit|ivfflat_.*)$'
+if [ $# -eq 0 ]; then
+  set -- $(cd "$SRC/test/sql" && ls *.sql | sed 's/\.sql$//' | grep -Ev "$SKIP_RE")
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)